ci/build-synapps-deps.sh

```diff
@@ -5,11 +5,12 @@ set -e
 mkdir external
 cd external
 
-wget -nv https://github.com/epics-modules/asyn/archive/R4-26.tar.gz
-tar -zxf R4-26.tar.gz
-echo "EPICS_BASE=/usr/lib/epics" > asyn-R4-26/configure/RELEASE
-#echo "EPICS_LIBCOM_ONLY=YES" >> asyn-R4-26/configure/CONFIG_SITE
-make -C asyn-R4-26/
+#wget -nv https://github.com/epics-modules/asyn/archive/R4-26.tar.gz
+#tar -zxf R4-26.tar.gz
+git clone https://github.com/epics-modules/asyn asyn-R4-31
```

Comment:
A simpler, and less github specific, alternative is to clone with ```--depth 10``` (or a similarly small number).  This can be combined with ```--branch <name>```.

wrt. travis-ci, this leds naturally to setting the branch name with an environment variable.  As for example https://github.com/epics-modules/devlib2/blob/master/.travis.yml where I build against several different Base versions.